crate/resources/crate.yml

```diff
@@ -811,29 +853,30 @@ discovery.zen.fd.ping_interval: 1s
 # enable/disable elasticsearch HTTP REST API
 #es.api.enabled: false
 
-###################################### Blob ####################################
-
-# Path to directory where to store blob data allocated for this node.
-# By default blobs will be stored under the same path as normal data.
-# A relative path value is relative to CRATE_HOME.
-#
-#blobs.path: /path/to/blobs
-
 
-########################## BACKUP / RESTORE ##########################
+############################# BACKUP / RESTORE ###############################
 
 # Paths where Repositories of type fs may be created:
-#
 #path.repo: /path/to/shared/fs,/other/shared/fs
 
-
 # Whitelist for urls that can be used with the url repository type
 # Wildcards are supported in the host, path, query and fragment parts.
 #
 # Supported protocols are: "http", "https", "ftp", "file" and "jar"
 # While only "http", "https" and "ftp" need to be listed here for usage in
 # URL repsoitories.
-#
 # "file" urls must be prefixed with an entry configured in ``path.repo``
-#
 #repositories.url.allowed_urls: ["http://example.org/root/*", "https://*.mydomain.com/*?*#*"]
+
+
+###################### POSTGRES WIRE PROTOCOL SUPPORT ########################
+
+# CrateDB supports the PostgreSQL wire protocol v3 and emulates a PostgreSQL
+# server v9.5. For further information read our documentation:
+# https://crate.io/docs/reference/en/latest/protocols/postgres.html
+#
+# This allows you to connect to CrateDB using one of the many Postgres tools
+# and libraries
+#psql.enabled: true
+#psql.port: 5432
```

Comment:
Uncomment and let's replace `5432` port with another one to avoid any conflicts from accidentaly running postgres or another CrateDB instance on the same machine.